backend/src/Designer/Configuration/PlatformSettings.cs

```diff
@@ -70,5 +70,20 @@ public class PlatformSettings : ISettingsMarker
         /// Value is stored as an environment variable in the AKS cluster.
         /// </summary>
         public string SubscriptionKeyHeaderName { get; set; }
+
+        /// <summary>
+        /// The Url used for calling the resourceregistry
+        /// </summary>
+        public string ResourceRegistryUrl { get; set; }
```

Comment:
Are these values used? Seems like something related to a only one environment setup. Is this not part of the REsourceRegistryBaseUrlSetting where there is a list?